File: nawanolza/server/src/main/java/ssafy/nawanolza/server/domain/entity/Member.java
package ssafy.nawanolza.server.domain.entity;

import lombok.*;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity @Getter
@Table(name = "members")
@EntityListeners(AuditingEntityListener.class)
@NoArgsConstructor(access = AccessLevel.PROTECTED) @AllArgsConstructor @Builder
public class Member {

    @Id
    @Column(name = "member_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    private String email;

    private String image;

    @CreatedDate
    @Column(name = "created_at", updatable = false)
    private LocalDateTime createdAt;
}

File: nawanolza/client/app/src/main/java/com/example/nawanolza/retrofit/Member.kt
package com.example.nawanolza.retrofit

import java.io.Serializable

data class Member(
    val createdAt: String,
    val email: String,
    val id: Int,
    val image: String,
    val name: String
): Serializable {
    constructor(): this("", "", 0, "", "")
}